.circleci/main.yml

```diff
@@ -1820,3 +1809,34 @@ commands:
       - pulumi/login
       - run: |
           pulumi plugin install resource eks v0.40.0
+  install-go-releaser:
+    steps:
+      - run:
+          name: install-go-releaser
+          command: |
+            mkdir -p /home/circleci/bin
+            wget https://github.com/goreleaser/goreleaser/releases/download/v<< pipeline.parameters.go-releaser-version >>/goreleaser_Linux_x86_64.tar.gz
+            tar zxvf goreleaser_Linux_x86_64.tar.gz -C /home/circleci/bin goreleaser
+            rm -rf goreleaser_Linux_x86_64.tar.gz
+            goreleaser -v
+  wait-for-docker:
```

Comment:
These are added but not used, at least in the PR. Is that ok? I thought these jobs (wait-for-docker and start-minikube) existed a while ago honestly, so I'm just not sure why they show as added in the diff.

Reply:
yea its how the `git cherry-pick` brought them. shouldn't hurt anything. if we need to layer any commit on top after for what ever reason rather have the commit stack as close to what it was, so decided to leave it